src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs

```diff
@@ -115,9 +115,9 @@ private static IHostBuilder EnableLogging(UpgradeOptions options, IHostBuilder h
 
                     services.AddSingleton<IUpgradeStateManager, FileUpgradeStateFactory>();
 
+                    services.AddExtensions(context.Configuration, options.Extension);
```

Comment:
Does the order here matter? I would've thought it didn't.

Reply:
Just so the messages of extensions are shown first